Lib9c/Model/State/GameConfigState.cs

```diff
@@ -71,6 +75,7 @@ public override IValue Serialize()
                 [(Text) "daily_arena_interval"] = DailyArenaInterval.Serialize(),
                 [(Text) "weekly_arena_interval"] = WeeklyArenaInterval.Serialize(),
                 [(Text) "required_appraise_block"] = RequiredAppraiseBlock.Serialize(),
+                [(Text) "battle_arena_interval"] = BattleArenaInterval.Serialize(),
```

Comment:
it can be occur `InvalidBlockStateRootHashException` beacuase serailze value is changed. this value serialize only interval > 0